packages/backend/src/app/authentication/authorization-middleware.ts

```diff
@@ -206,50 +206,21 @@ function getTableNameFromResource(resource: string | undefined): string | undefi
     return undefined
 }
 
-function isAuthenticatedRoute(routerPath: string, method: string): boolean {
+function isAuthenticatedRoute(fastifyRequest: FastifyRequest, routerPath: string): boolean {
```

Comment:
Any reason to include `routerPath` in `isAuthenticatedRoute` params? It can already be accessed using `request.routerPath`, right?